src/SignalR/clients/csharp/Client.Core/src/HubConnectionState.cs

```diff
@@ -13,8 +13,16 @@ public enum HubConnectionState
         /// </summary>
         Disconnected,
         /// <summary>
+        /// The hub connection is connecting.
+        /// </summary>
+        Connecting,
+        /// <summary>
         /// The hub connection is connected.
         /// </summary>
-        Connected
+        Connected,
```

Comment:
Is this considered a [breaking change](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#property-field-parameter-and-return-values) since it is changing the value of the `Connected` enum?

Reply:
This isn't actually changing the value of this enum. It only looks like a diff because we don't use [dangling commas](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)

Reply:
Sorry the comment is on this line, but the change is inserting the `Connecting` enum above `Connected` which makes `Connecting` = 1 and `Connected` = 2 when it was 1 previously.

Reply:
Good point! I think you're right

Reply:
It's definitely a breaking change. The question is, do we care more about breaking someone (if anyone) who might be treating HubConnection.State as an int? Or do we care more about having the enum values listed in a logical order?

I could be convinced either way.

Reply:
Use the natural ordering and just file an announcement

Reply:
I'd rather just preserve the ordering. Yes, we do breaking changes in major versions, but only ones that provide value. Adding a new enum variant should be done in a non-breaking way IMO. Just tack this on to the end instead of in the middle.